docs/error_codes.md

```diff
@@ -0,0 +1,29 @@
+# Error Codes
+
+This document is suggesting that a high level error category 6xxx(can be another number if 6xxx is reserved) be used to represent errors that are returned to a third party such as PISP’s and AISP’s. These are error codes the Switch would send back to a PISP.
+
+- **Third Party Error** -- **60**_xx_
+
+| **Error Code** | **Name** | **Description** | /parties | /thirdPartyRequest | /consentRequests | /authorizations |
```

Comment:
Is it thirdPartyRequest or thirdPartyRequests ? typo or intentional?

Reply:
thirdPartyRequests. typo > - <